.github/workflows/ci.yml

```diff
@@ -121,6 +125,7 @@ jobs:
         uses: "networktocode/gh-action-setup-poetry-environment@v4"
         with:
           python-version: "${{ matrix.python-version }}"
+          poetry-install-options: "--with dev"
```

Comment:
Same as above

Reply:
I need to install the library to test it, and the default only install the dev, same happened in mypy